File: 3828/3828.c
int finalElementAfterSubarrayDelet(struct TreeNode* root) {
    if (!root) return 0;
    int l = finalElementAfterSubarrayDelet(root->left);
    int r = finalElementAfterSubarrayDelet(root->right);
    return 1 + (l > r ? l : r);
}

File: 3828/3828.go
func finalElementAfterSubarrayDelet(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := finalElementAfterSubarrayDelet(root.Left)
	r := finalElementAfterSubarrayDelet(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}